internal/store/fleet.go

```diff
@@ -23,6 +23,8 @@ type Fleet interface {
 	Create(ctx context.Context, orgId uuid.UUID, fleet *api.Fleet, callback FleetStoreCallback) (*api.Fleet, error)
 	Update(ctx context.Context, orgId uuid.UUID, fleet *api.Fleet, callback FleetStoreCallback) (*api.Fleet, error)
 	List(ctx context.Context, orgId uuid.UUID, listParams ListParams, opts ...ListOption) (*api.FleetList, error)
+	ListRolloutDeviceSelection(ctx context.Context, orgId uuid.UUID) (*api.FleetList, error)
+	ListDisruptionAllowanceFleets(ctx context.Context, orgId uuid.UUID) (*api.FleetList, error)
```

Comment:
Same question as above about these - what's missing from ListFleets?

Reply:
These use capabilities that are missing in ListFleets:
The first tries to bring fleets that have devices that have not been rolled out (where not been processed by the fleet-rollout task).
The second tries to bring fleets that have devices that have not been rendered yet.
In general,  ListFleets cannot use a filter which is based on the underlying device population.